### PR TITLE
Ped and Add PPP

### LINKED ISSUE #202: [PE-D] Deleting/Editing similar ingredients
Building on Issue number 3, if there are multiple instances of an ingredient in the fridge, as shown, 

using the command deleteF 3 causes the item at index 2 to be deleted instead.

The same thing happens with the editF command.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: PrestonTYR/ped#4

### LINKED ISSUE #208: [PE-D] Possible feature bug: if I add a duplicate ingredient into the fridge, shouldn't it just accumulate in the fridge?
As per title. E.g. if I want to add the same thing twice, it should jus accumulate I think. 

current fridge:

new addition: 

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: rtshkmr/ped#1

### LINKED ISSUE #226: [PE-D] Duplicate Fridge ingredients
Small changes in the name and quantity allows same ingredients to be added.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: todoge/ped#7

### LINKED ISSUE #236: Make sure no duplicate ingredients in addR (check ordering)

### PR DESCRIPTION
**Pull request by oliviajohansen**

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/WishfulShrinkingParserTest.java [hunk 1]
```diff
@@ -125,14 +129,25 @@ public void parseCommand_selectRecipe() throws Exception {
...
-    //@Test
-    //public void parseCommand_edit() throws Exception {
-    //    Recipe recipe = new RecipeBuilder().build();
-    //    EditRecipeCommand.EditRecipeDescriptor descriptor = new EditRecipeDescriptorBuilder(recipe).build();
-    //    EditRecipeCommand command = (EditRecipeCommand) parser.parseCommand(EditRecipeCommand.COMMAND_WORD + " "
-    //            + INDEX_FIRST_RECIPE.getOneBased() + " " + RecipeUtil.getEditRecipeDescriptorDetails(descriptor));
-    //    assertEquals(new EditRecipeCommand(INDEX_FIRST_RECIPE, descriptor), command);
-    //}
+//    @Test
+//    public void parseCommand_editRecipe() throws Exception {
```
Comment (caitlinjee) [inline src/test/java/seedu/address/logic/parser/WishfulShrinkingParserTest.java]: dont forget to remove this

File: src/test/java/seedu/address/testutil/RecipeUtil.java [hunk 2]
```diff
@@ -72,15 +72,15 @@ public static String getEditRecipeDescriptorDetails(EditRecipeDescriptor descrip
...
+                                .map(item -> item.getValue() + " " + PREFIX_QUANTITY + item.getQuantity())
                                 .reduce("", (a, b) -> b.equals("") ? a : b + ", " + a)).append(" "));
         descriptor.getInstruction()
                 .ifPresent(instr -> sb.append(PREFIX_INSTRUCTION)
                         .append(instr.stream()
                                 .map(item -> item.toString() + ". ")
                                 .reduce("", (a, b) -> a + b).trim()).append(" "));
-        descriptor.getInstruction().ifPresent(instr -> sb.append(PREFIX_INSTRUCTION).append(" "));
-        descriptor.getRecipeImage().ifPresent(img -> sb.append(PREFIX_RECIPE_IMAGE).append(" "));
+//        descriptor.getInstruction().ifPresent(instr -> sb.append(PREFIX_INSTRUCTION).append(" "));
```
Comment (caitlinjee) [inline src/test/java/seedu/address/testutil/RecipeUtil.java]: dont forget to remove also

File: src/test/java/seedu/address/logic/parser/WishfulShrinkingParserTest.java (hunk 1 above)
Comment (oliviajohansen) [reply src/test/java/seedu/address/logic/parser/WishfulShrinkingParserTest.java]: this test im waiting for jq's stuff which will fix it

File: src/test/java/seedu/address/testutil/RecipeUtil.java (hunk 2 above)
Comment (oliviajohansen) [reply src/test/java/seedu/address/testutil/RecipeUtil.java]: ok

**Review by caitlinjee — APPROVED** [review]

**Review by jiaqi20 — APPROVED** [review]
lgtm

**Review by daongochieu2810 — APPROVED** [review]
``

**Review by TianYong-Goh — APPROVED** [review]
lgtm

**Pull request merged**